js/ui/handler/drag_rotate.js

```diff
@@ -200,7 +200,14 @@ class DragRotateHandler {
         } else {
             const buttons = (e.ctrlKey ? 1 : 2),  // ? ctrl+left button : right button
                 button = (e.ctrlKey ? 0 : 2);   // ? ctrl+left button : right button
-            return (e.type === 'mousemove' ? e.buttons & buttons === 0 : e.button !== button);
+            let eventButton = e.button;
+            if (typeof InstallTrigger !== 'undefined' && e.button === 2 && e.ctrlKey) {
```

Comment:
👍 just confirmed that this bug is not present on Windows FF, so I'll add this case check